client/lcd/lcd_test.go

```diff
@@ -630,14 +631,17 @@ func doBond(t *testing.T, port, seed, name, password string, delegatorAddr, vali
 		"account_number": %d,
 		"sequence": %d,
 		"gas": 10000,
-		"delegate": [
+		"delegations": [
 			{
 				"delegator_addr": "%s",
 				"validator_addr": "%s",
 				"bond": { "denom": "%s", "amount": 60 }
 			}
 		],
-		"unbond": []
+		"begin_unbondings": [], 
+		"complete_unbondings": [], 
+		"begin_redelegates": [], 
+		"complete_redelegates": []
```

Comment:
`chain_id` might be needed here if not specified already (can't see entire JSON struct in diff)

Reply:
added, using viper to receive in the test (which needs to be changed), I think that'll be my next PR, refactor out viper usage, pretty much the same as using global variables